src/portfolio.js

```diff
@@ -126,21 +126,25 @@ const workExperiences = {
       company: "Facebook",
       companylogo:  require("./assets/images/facebookLogo.png"),
       date: "June 2018 – Present",
-      desc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+      desc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      descBullets: [
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+      ]
     },
     {
       role: "Front-End Developer",   
       company: "Quora",
       companylogo: "https://www.referralcandy.com/wp-content/uploads/2017/06/quora-featured-image-2-740x740.png",
```

Comment:
Could you get the png and save it in the assets folder so the app serves it locally like the other two logos.

Reply:
@kartik918 Done